src/main/java/com/sfuentes/Farm/Farm.java

```diff
@@ -0,0 +1,30 @@
+package com.sfuentes.Farm;
+
+import java.util.List;
+
+public class Farm {
+
+  public Farm() {
+  }
+
+  public double getQualityByHen(Hen hen) {
+    return hen.getHenWeight() * hen.getHenHeight() / hen.getNumberOfEggs();
+  }
+
+  public double getEggPriceByKilo(Hen hen) {
```

Comment:
Rename this method to getEggPricePerKilo

Reply:
fix